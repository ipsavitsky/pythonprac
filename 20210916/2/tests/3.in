[7, 16, 4, 5, 2, 7]
