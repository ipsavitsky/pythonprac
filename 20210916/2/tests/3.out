[2, 4, 5, 7, 7, 16]
